### ISSUE TITLE
Afriso AVD 20‐D Thermostat not recognized.

### ISSUE THREAD

**Issue by Taraman17** [Taraman17/hass-homee#77]
## Type of problem

- [ ] I have a question.
- [ ] A feature does not work as expected.
- [X] I found a bug (see error log below).

## Version
2.9.3

## Detailed description
The Device is causing an error when being set up

## Error log
```text
2024-05-11 08:33:42.420 ERROR (MainThread) [homeassistant.components.climate] Error adding entity climate.heizung_schlafzimmer for domain climate with platform homee
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 580, in _async_add_entities
    await coro
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 892, in _async_add_entity
    await entity.add_to_platform_finish()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1359, in add_to_platform_finish
    self.async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1009, in async_write_ha_state
    self._async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1132, in _async_write_ha_state
    state, attr, capabilities, shadowed_attr = self.__async_calculate_state()
                                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1067, in __async_calculate_state
    state = self._stringify_state(available)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1015, in _stringify_state
    if (state := self.state) is None:
                 ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/climate/__init__.py", line 324, in __getattribute__
    return super().__getattribute__(__name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/climate/__init__.py", line 455, in state
    hvac_mode = self.hvac_mode
                ^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/climate/__init__.py", line 324, in __getattribute__
    return super().__getattribute__(__name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/homee/climate.py", line 117, in hvac_mode
    if self.get_attribute(AttributeType.HEATING_MODE).current_value == 0:
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/homee/__init__.py", line 363, in get_attribute
    return self._node.get_attribute_by_type(attribute_type)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/pymee/model.py", line 265, in get_attribute_by_type
    return self._attribute_map[type]
           ~~~~~~~~~~~~~~~~~~~^^^^^^
KeyError: 258
```

**Comment by Taraman17** [comment]
The error is caused, because the integration expects devices with profile 3006 / RADIATOR_THERMOSTAT to have an attribute of Type 258 = HEAT_MODE
Since the Afriso Thermostat deos not have this, the setup fails: https://github.com/Taraman17/hass-homee/wiki/Afriso-AVD-20%E2%80%90D

**Comment by Milobaby1** [comment]
Das heißt sie funktionieren in hA nicht!

**Comment by Taraman17** [comment]
Closed with release 2.9.5
